tests/integration/components/sl-radio-group-test.js

```diff
@@ -0,0 +1,165 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent( 'sl-radio-group', 'Integration | Component | sl radio group', {
+    integration: true
+});
+
+test( 'The disabled state applies the disabled attribute and class', function( assert ) {
+    this.set( 'testName', 'test' );
```

Comment:
Assertion Failed: The name property must be set on the sl-radio-group component. The name property is required.